sdk/core/abort-controller/package.json

```
@@ -7,15 +7,18 @@
   "module": "dist-esm/src/index.js",
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:es6": "tsc -p tsconfig.json",
-    "build:nodebrowser": "rollup -c 2>&1",
     "build:types": "downlevel-dts types/src types/3.1",
-    "build:test": "rollup -c rollup.test.config.js 2>&1",
-    "build": "npm run build:es6 && npm run build:nodebrowser && npm run build:types",
+    "build:ts": "tsc -p .",
+    "build:node": "npm run build:ts && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run extract-api",
+    "build:browser": "npm run build:ts && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:test": "npm run build:test:node && npm run build:test:browser",
+    "build:test:node": "npm run build:ts && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:browser": "npm run build:ts && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
+    "build": "npm run build:ts && npm run build:node && npm run build:browser && npm run build:types",
```

```suggestion
    "build": "npm run build:ts && npm run build:node && npm run build:browser && npm run build:types",
```

This calls `npm run build:ts` thrice which would be very bad for the CI if we do it for each package.
Please update the build command to not have repetitions.

---

Fixed